Libraries/Azure.psm1

```diff
@@ -1382,6 +1392,27 @@ Function Invoke-AllResourceGroupDeployments($SetupTypeData, $CurrentTestData, $R
 			Add-Content -Value "$($indents[4])}," -Path $jsonFile
 			#endregion
 
+			#region Security Profile
+			if ($CurrentTestData.SetupConfig.SecureBoot -or $CurrentTestData.SetupConfig.vTPM) {
+				Add-Content -Value "$($indents[4])^securityProfile^: " -Path $jsonFile
+				Add-Content -Value "$($indents[4]){" -Path $jsonFile
+				Add-Content -Value "$($indents[5])^uefiSettings^: " -Path $jsonFile
+				Add-Content -Value "$($indents[5]){" -Path $jsonFile
+				if ($CurrentTestData.SetupConfig.SecureBoot -and !$CurrentTestData.SetupConfig.vTPM) {
```

Comment:
Should we check if both SecureBoot and vTPM are true condition? This is not correct configuration but we should check this faulty case in if-statement.

Reply:
No, this is by design.  it's correct configuration even use only SecureBoot or vTPM.